samples/bluetooth/hci_pwr_ctrl/src/main.c

```diff
@@ -22,6 +22,9 @@
 #include <zephyr/bluetooth/gatt.h>
 #include <zephyr/bluetooth/services/hrs.h>
 
+BUILD_ASSERT(IS_ENABLED(CONFIG_BT_HAS_HCI_VS),
```

Comment:
Unrelated to this PR, but there is actually somewhat of a prefix name clash between `BT_HAS_HCI_VS` and `BT_HAS` (Hearing Access Service)

Reply:
Good point. I think it'd make sense to swap HAS and HCI, i.e. make it `BT_HCI_HAS_VS_EXT`. But yes, something for a follow-up PR.

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/81851